ScipDotnet/ScipProjectIndexer.cs

```diff
@@ -150,6 +152,12 @@ await host.Services.GetRequiredService<MSBuildWorkspace>()
                 var walker = new ScipVisualBasicSyntaxWalker(symbolFormatter, semanticModel);
                 walker.Visit(root);
             }
+            else
+            {
+                Logger.LogWarning(
+                    "Skipping document {DocumentFilePath} because it has language {DocumentLanguage} and scip-dotnet currently only supports C# and Visual Basic.",
+                    document.FilePath, language);
+            }
```

Comment:
Do we get automatic warning suppression somehow? If not, please record this kind of warning in a "limited array" type (where you can push a fixed number of elements, and further attempts to push just bump a counter) instead of emitting it on the fly.

Here's an example of the kind of API you can define: https://sourcegraph.sourcegraph.com/github.com/sourcegraph/scip_v2_demo/-/blob/fill_db/scipext/context.go?L20:27-20:33#tab=references

I'm afraid that if a project has mixed C#-F#, then trying to index code should not trigger thousands of repetitive warnings which the user has to filter out. Instead, we should have just one warning with a handful of examples (e.g. say 3-5) as well as a count of how many files were skipped.

Reply:
Thanks for the feedback, this led me to validate and this can never be hit. Natively .net will not support cross-languages in a project. Removed this log